lib/rules/no-v-text-v-html-on-component.js

```diff
@@ -15,21 +15,50 @@ module.exports = {
       url: 'https://eslint.vuejs.org/rules/no-v-text-v-html-on-component.html'
     },
     fixable: null,
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          allow: {
+            type: 'array',
+            items: {
+              type: 'string'
+            },
+            uniqueItems: true
+          }
+        },
+        additionalProperties: false
+      }
+    ],
     messages: {
       disallow:
         "Using {{directiveName}} on component may break component's content."
     }
   },
   /** @param {RuleContext} context */
   create(context) {
+    /** @type {string[]} */
+    let allowedComponent = []
+    if (context.options[0] && context.options[0].allow) {
+      allowedComponent = [...context.options[0].allow]
+    }
+
+    /**
+     * Check whether the given node is an allowed component or not.
+     * @param {VElement} node The start tag node to check.
+     * @returns {boolean} `true` if the node is an allowed component.
+     */
+    function isAllowedComponent(node) {
+      return allowedComponent.includes(node.rawName)
```

Comment:
This should also allow `NuxtLink` if `nuxt-link` is in the `allow` list, and vice-versa. Please also add test cases for that and adjust the documentation.

See #2314 for a similar implementation.

Reply:
thanks for the reference, that makes sense!